inits Deployment contrib (#2312)
open-telemetry/opentelemetry.io: content/en/docs/collector/deployment/_index.md

---
title: Deployment
description: Patterns you can apply to run the OpenTelemetry collector
weight: 2
---

> The OpenTelemetry collector consists of a single binary which you can use in
> different ways, for different use cases. This section describes deployment
> patterns, their use cases along with pros and cons and best practices for
> collector configurations for cross-environment and multi-backend deployments.

Reviewer: nit: make the first sentence structure simpler
```suggestion
You can deploy the OpenTelemetry collector in different ways depending on your 
use case. This section describes deployment
patterns, their use cases along with pros and cons and best practices for
collector configurations for cross-environment and multi-backend deployments.
```

> ## Other information

Reviewer 2: One question that has come up fairly frequently regarding collector deployments is "when/why should I use the [Collector Operator](https://github.com/open-telemetry/opentelemetry-operator)?"

Would be helpful for end-users if the intro mentioned how the operator (optionally) fits into k8s-based deployments.

Reviewer 3: @mhausenblas, wdyt? I am always a fan of having inter-linked pages to point people to other resources (not the repo, but the docs for k8s operator at /docs/k8s-operator/), but I don't see yet how to incorporate that to this page.

> - GitHub repo [OpenTelemetry Collector Deployment Patterns][gh-patterns]

Reviewer: ```suggestion
- [Repository with full configuration examples for different deployment patterns][gh-patterns]
```

> - YouTube video [OpenTelemetry Collector Deployment Patterns][y-patterns]

Reviewer: I would make it clear that this is a talk
```suggestion
- KubeCon NA 2021 Talk ["OpenTelemetry Collector Deployment Patterns"][y-patterns]
```

Reviewer 4: Good suggested change, but drop the quotes.

Reviewer: edited, thanks for the feedback :)
